src/Fmt.ml

```diff
@@ -164,13 +164,16 @@ let box_depth_colors = [|32; 33; 94; 31; 35; 36|]
 let box_depth_color () =
   box_depth_colors.(!box_depth % Array.length box_depth_colors)
 
-let debug_box_open box_kind n fs =
+let debug_box_open ?name box_kind n fs =
   if !box_debug_enabled then (
     let openning =
       if n = 0 then box_kind else Format.sprintf "%s<%d" box_kind n
     in
+    let name =
+      Option.value_map name ~f:(fun s -> Format.sprintf "<%s>" s) ~default:""
```

Comment:
This will look like `[hv<name>>` or `[hv<2<name>>`.
What do you think of simply using a single char (eg. space or `:`), so it looks like `[hv name>` or `[hv<2 name>` ?

Reply:
Fixed